### PR TITLE
docs: integrate supabase in dhiwise, steps

### PR DESCRIPTION
**Pull request by saloni137**
## What kind of change does this PR introduce?

A new documentation with steps

### CODE REVIEW

**Comment by thorwebdev** [comment]
Thanks @saloni137 🙌  sorry for the delay, this one feel through the cracks. Does that mean https://docs.dhiwise.com/knowledgehub/mobile-app-builders/supabase-integration/supabase-integration-steps has been deprecated? Or has it been moved somewhere else?

Anyway, I can point https://supabase.com/partners/dhiwise to point to this new documentation of that's preferred?

File: apps/reference/docs/guides/integrations/dhiwise.mdx [hunk 1]
```diff
@@ -0,0 +1,175 @@
+---
+id: dhiwise
+title: 'DhiWise'
+description: 'Get started with Supabase and DhiWise, Covert your Figma design into Flutter apps, Authenticate Users with Supabase and Connect Supabase as a backend in your Flutter application using DhiWise'
```
Comment (thorwebdev) [inline apps/reference/docs/guides/integrations/dhiwise.mdx:4]:
```suggestion
description: 'Get started with Supabase and DhiWise. Convert your Figma designs into Flutter apps, store data and authenticate your users.'
```

File: apps/reference/docs/guides/integrations/dhiwise.mdx [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+---
+import select from "/img/guides/integrations/dhiwise/select.png"
+import create from "/img/guides/integrations/dhiwise/create.png"
+
+This guide explains how to quickly connect Supabase back-end to an DhiWise Flutter application.
+
+[DhiWise](https://www.dhiwise.com/) is a Developer tools to covert Figma designs into React and Flutter applications. It let's you integrate Database and APIs to your React and Flutter Apps quickly.
```
Comment (thorwebdev) [inline apps/reference/docs/guides/integrations/dhiwise.mdx:11]:
```suggestion
[DhiWise](https://www.dhiwise.com/) is a Developer tool to convert Figma designs into React and Flutter applications. It lets you integrate Database and APIs into your React and Flutter Apps quickly.
```

File: apps/reference/docs/guides/integrations/dhiwise.mdx [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+If you don’t have an DhiWise account, create one [here](https://app.dhiwise.com).
```
Comment (thorwebdev) [inline apps/reference/docs/guides/integrations/dhiwise.mdx:13]:
```suggestion
If you don’t have a DhiWise account, create one [here](https://app.dhiwise.com).
```

File: apps/reference/docs/guides/integrations/dhiwise.mdx [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+
+## step 3: Find the API key and URL
+
+Once your project is created you can access the API Key and URL string, Or if you already have an account go to your `organization-> app-> settings-> API`.
+
+![auth keys](/img/guides/integrations/dhiwise/authKeys.png)
+
+## Step 4: Integrations
+
+Now, There are two ways you can integrate Supabase into DhiWise Flutter Application.
```
Comment (thorwebdev) [inline apps/reference/docs/guides/integrations/dhiwise.mdx:42]:
```suggestion
There are two ways you can integrate Supabase into your DhiWise Flutter applications.
```

**Review by thorwebdev — CHANGES_REQUESTED** [review]
Thanks for this. Just some small nits :)

**Comment by saloni137** [comment]
@thorwebdev Yes, the New Doc is here: https://docs.dhiwise.com/docs/flutter/supabase-integration

**Comment by saloni137** [comment]
@thorwebdev Applied your changes!

**Review by thorwebdev — APPROVED** [review]
LGTM, thanks

**Comment by saloni137** [comment]
@thorwebdev, why the page still has grammar errors even tho I committed your suggested changes?

**Pull request merged**